account_multiple_account_type_reconciliation/static/src/js/account_multiple_account_type_reconciliation.js

```diff
@@ -0,0 +1,32 @@
+//-*- coding: utf-8 -*-
+//© 2017 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+openerp.account_multiple_account_type_reconciliation = function(instance) {
+	instance.web.account.bankStatementReconciliationLine.include({
+
+		selectMoveLine : function(mv_line) {
```

Comment:
@daramousk Thanks for the explanation, please put it also as comments (maybe withouth the exact linenumber, just point out the method) in the code. Is there no back-end check on reconciling debit and credit, or is it all in the frontend?